hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3ABucketExistence.java

```diff
@@ -189,6 +190,17 @@ public void testAccessPointRequired() throws Exception {
         () -> FileSystem.get(uri, configuration));
   }
 
+  /**
+   * Create a configuration which has bucket probe 2 and the endpoint.region
+   * option set to "eu-west-1" to match that of the ARNs generated.
+   * @return a configuration for tests which are expected to fail in specific ways.
+   */
+  private Configuration createArnConfiguration() {
+    Configuration configuration = createConfigurationWithProbe(2);
+    configuration.set(AWS_REGION, "eu-west-1");
+    return configuration;
+  }
```

Comment:
This util ain't very generic, could have taken ``region`` as argument and changed the method name to be like ``create*WithRegion(String region)``, so could have used it for different purpose in future as well. 

Juzz my random thoughts, no need to change :)

Reply:
noted